consensus/hotstuff/committees/consensus_committee.go

```diff
@@ -11,35 +11,83 @@ import (
 	"github.com/onflow/flow-go/model/flow"
 	"github.com/onflow/flow-go/model/flow/filter"
 	"github.com/onflow/flow-go/state/protocol"
-	"github.com/onflow/flow-go/state/protocol/seed"
 )
 
-var errSelectionNotComputed = fmt.Errorf("leader selection for epoch not yet computed")
+// ErrViewForUnknownEpoch is returned when a by-view query is made with a view
+// outside all cached epochs. This can happen when a query is made for a view in the
+// next epoch, if that epoch is not committed yet. This can also happen when an
+// old epoch is queried (>3 in the past), even if that epoch does exist in storage.
+var ErrViewForUnknownEpoch = fmt.Errorf("by-view query for unknown epoch")
+
+// staticEpochInfo contains leader selection and the initial committee for one epoch.
+type staticEpochInfo struct {
```

Comment:
Is the `staticEpochInfo` supposed to be an immutable data structure? If so, it would be great to explicitly point this out in the goDoc.

Reply:
Added to godoc